packages/editor/src/lib/editor/shapes/BaseBoxShapeUtil.tsx

```diff
@@ -27,4 +27,12 @@ export abstract class BaseBoxShapeUtil<Shape extends TLBaseBoxShape> extends Sha
 			points: this.getGeometry(shape).bounds.cornersAndCenter,
 		}
 	}
+
+	override getInterpolatedProps(startShape: Shape, endShape: Shape, t: number): Shape['props'] {
+		return {
+			...endShape.props,
+			w: startShape.props.w + (endShape.props.w - startShape.props.w) * t,
+			h: startShape.props.h + (endShape.props.h - startShape.props.h) * t,
```

Comment:
we have a `lerp` in `@tldraw/utils`
```suggestion
			w: lerp(startShape.props.w, endShape.props.w, t),
			h: lerp(startShape.props.h, endShape.props.h, t),
```